Makefile

```diff
@@ -19,7 +19,7 @@ tag:
 
 # Builds “Aegis Safe” into a binary.
 safe-build:
-	./hack/safe-build.sh "aegis-ist-safe"
+	./hack/build.sh "aegis-ist-safe" "./app/safe/cmd/main.go"
```

Comment:
consolidated things to a single `build.sh`; that resulted in an extra parameter to be passed.